packages/aws-cdk-lib/aws-ses/lib/email-identity.ts

```diff
@@ -419,6 +434,16 @@ export class EmailIdentity extends Resource implements IEmailIdentity {
       });
     }
 
+    if (props.autoDmarc && props.identity.hostedZone) {
```

Comment:
If `props.dmarcReportEmail` is set but `props.autoDmarc` is not, then the email will be ignored, which I argue is unexpected. Instead, we should assume that `autoDmarc` is true if `dmarcReportEmail` is set.